app/domain/secureMessage/Request.scala

```diff
@@ -62,34 +62,38 @@ object Request {
   }
 
   private def contents(params: Params, company: String): List[Content] = {
+
+    val en = DateRange(params, englishLangKey)
+    val cy = DateRange(params, welshLangKey)
+
     params.accountType match {
       case "DutyDefermentStatement" =>
         List(
-          Content(englishLangKey, SubjectDutyDef,
-            encodeToUTF8Charsets(DutyDefermentBody(company, DateRange(params, englishLangKey), englishLangKey))),
-          Content(welshLangKey, SubjectDutyDefCy,
-            encodeToUTF8Charsets(DutyDefermentBody(company, DateRange(params, welshLangKey), welshLangKey))))
+          Content(englishLangKey, SubjectDutyDef + en.dateAsNumber,
```

Comment:
Please use interpolation instead of using +. Same for other places when + is being used to concatenate